index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
```

Comment:
I think mandating certain registries in the charter is quite a strong language. I suggest language below would be sufficient
```
The Working Group may will publish and maintain the following registries:
```

Reply:
I think you might be advocating for a may here? your suggested text has both may and will, if so if you agree with the intent of the registries why relax the language to a may?

Reply:
oops, thanks for catching this. I was advocating for `may` and also removing `at least` that mandates working on the below four registries. (edited the suggestion in the initial comment)